templates/webapi/layouts/navbar.html.ep

```diff
@@ -42,10 +44,12 @@
             </li>
         % if (current_user) {
             <li class="nav-item dropdown" id="user-action">
-                <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" >Logged in as <%= current_user->name %></a>
+                <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button"
+                   aria-haspopup="true" aria-expanded="false"
+                   title="Contains the Activity View and various configuration pages">Logged in as <%= current_user->name %></a>
                 <div class="dropdown-menu">
                   %= tag 'h3' => class => 'dropdown-header' => 'Operators Menu'
-                  %= link_to 'Activity View' => url_for('activity_view') => class => 'dropdown-item' => id => 'activity_view'
+                  %= link_to 'Activity View' => url_for('activity_view') => class => 'dropdown-item' => id => 'activity_view', title => 'Gives you an overview of your current jobs'
```

Comment:
Maybe `Provides an overview of your current jobs` sounds better? Feel free to resolve this if you think it does not.

Reply:
After a quick research I think both verbs are fine and *give* and overview is probably even the more commonly used verb (with overview).